recipes/dotnetcore-sdk/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "dotnetcore-sdk" %}
+{% set version = "3.1.100" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://dot.net/v1/dotnet-install.sh  # [not win]
+    sha256: 7cf038c274862c160c2fe8cfc0ea7471b48d1ac05cb201b7717d53354ccd139c  # [not win]
+  - url: https://dot.net/v1/dotnet-install.ps1  # [win]
+    sha256: 71a888f99a73936de8cbcc494df2706f77a69a98a9962a0351052c350f95be39  # [win]
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: False
+  # .NET assemblies are technically PE files with a small stub that links to
+  # mscoree.dll. That link exists as a compatibility shim for older versions of
+  # Windows, and should be ignored when determining the actual linking requirements
+  # for the .NET Core SDK.
+  missing_dso_whitelist:
+  - "C:\\Windows\\System32\\mscoree.dll"
+
+requirements:
+  build:
+  - {{ compiler('c') }}
+
+test:
+  commands:
+  - dotnet --version
+
+about:
+  home: https://github.com/dotnet/core-sdk
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: '.NET Core is a cross-platform version of .NET for building websites, services, and console apps.'
+
+  description: |
+    .NET Core is a free and open-source, managed computer software framework for Windows, Linux, and macOS operating systems.
+  doc_url: https://simplejson.readthedocs.io/
+  dev_url: https://github.com/simplejson/simplejson
```

Comment:
These two URLs are wrong

Reply:
Absolutely, thanks for catching that! Copied and pasted for the tutorial and forgot to change those two.